src/katgpucbf/fgpu/recv.py

```diff
@@ -256,9 +256,7 @@ def make_sensors(sensor_timeout: float) -> aiokatcp.SensorSet:
             TimeoutSensorStatusObserver(sensor, sensor_timeout, aiokatcp.Sensor.Status.NOMINAL)
             sensors.add(sensor)
 
-        sensors.add(
-            DeviceStatusSensor(sensors, "rx.device-status", "F-engine is receiving a good, clean digitiser stream")
-        )
```

Comment:
The one time I view the PR in GitHub and not in a text editor is obviously the one time an actual change needed to be addressed. Please and thank yous all around.